examples/kurth/input_kurth_10nC.in

```diff
@@ -0,0 +1,41 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000
+#beam.npart = 100000 #optional for increased precision
+beam.units = static
+beam.energy = 2.0e3
+beam.charge = 1.0e-8
+beam.particle = proton
+beam.distribution = kurth6d
+beam.sigmaX = 1.2154443728379865788e-3
+beam.sigmaY = 1.2154443728379865788e-3
+beam.sigmaT = 4.0956844276541331005e-4
+beam.sigmaPx = 8.2274435782286157175e-4
+beam.sigmaPy = 8.2274435782286157175e-4
+beam.sigmaPt = 2.4415943602685364584e-3
+
+
+###############################################################################
+# Beamline: lattice elements and segments
+###############################################################################
+lattice.elements = constf1
+lattice.nslice = 50
+#lattice.nslice = 100 #optional for increased precision
+
+constf1.type = constf
+constf1.ds = 2.0
+constf1.kx = 1.0
+constf1.ky = 1.0
+constf1.kt = 1.0
+
+
+###############################################################################
+# Algorithms
+###############################################################################
+algo.particle_shape = 2
+algo.space_charge = true
+
+amr.n_cell = 48 48 40
+#amr.n_cell = 72 72 72 #optional for increased precision
+geometry.prob_relative = 1.0
```

Comment:
The above test, "input_kurth_10nC.in", uses only a Kurth distribution and a constant focusing element (with space charge), features that are already tested elsewhere.  I suggest that this test can be removed.  The two important Kurth tests are "input_kurth_periodic.in" (without space charge) and "input_kurth_10nC_periodic.in" (with space charge).

Reply:
I agree - thanks for the update, I will add one more push that removes `input_kurth_10nC.in` :)